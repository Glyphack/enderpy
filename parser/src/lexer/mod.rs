mod diagnostics;
pub mod fstring;
pub mod lexer;
