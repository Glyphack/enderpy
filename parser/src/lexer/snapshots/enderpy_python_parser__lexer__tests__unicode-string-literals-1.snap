---
source: parser/src/lexer/mod.rs
description: "u\"world\""
---
[
    Token {
        kind: Unicode,
        value: Str(
            "u\"world\"",
        ),
        start: 0,
        end: 8,
    },
]
